styles/globals.less

```diff
@@ -0,0 +1,71 @@
+// Variables
+
+// colors
+@background: #FAFAFA;
```

Comment:
Can you move color variables to a separate file? I would name it 'variables.less'. To use them in other less files you will need to do something like:

```
@import 'styles/globals.less'
```

This means that you would also import general styles from below (body etc), it would be cleaner just to keep variables and other reusable styles in a single separate file

Reply:
Ok, Thank you